django-backend/fecfiler/settings/production.py

```diff
@@ -0,0 +1,18 @@
+from .base import *  # noqa F401, F403
```

Comment:
This is still getting flagged in SonarCloud. I think their decorator to skip linting, and all other static checks on the line, is "# NOSONAR"
Not sure, but maybe you can stack the noqa with the NOSONAR like:
`# NOSONAR noqa F401, F403`

Reply:
Fixed, thank you!